go/pio/host/internal/internal.go

```diff
@@ -20,14 +20,17 @@ var (
 	osRelease map[string]string
 )
 
-func readAndSplit(path string) map[string]string {
-	bytes, err := ioutil.ReadFile("/proc/cpuinfo")
+// readAndSplit reads the file at path, splits each line at sep, and places each line
```

Comment:
I fixed it differently in 7c1944a9ebadc50eb39e1462f48782c0402991ea and added unit test, so you can rebase and skip this part.